tox.ini

```diff
@@ -7,8 +7,8 @@
 skipsdist = True
 usedevelop = True
 envlist =
-    py{38,39,310}-dj{32,41}-wt{41,42,50}
-    py{311}-dj{41,42}-wt{50}
+    py{38,39,310,311}-dj{32,41}-wt{41,42,50}
+    py{39,310,311}-dj{42}-wt{51}
```

Comment:
Seeing that dj32 isn't supported on py311 can the matrix be adjusted please.

Reply:
How about this?
https://github.com/torchbox-forks/wagtail-django-recaptcha/pull/8/commits/5282c47a8b2a08fc91909ea8e0d265c1f67ee0a2